source/frontend/src/hooks/pims-api/useApiDocuments.ts

```diff
@@ -77,10 +77,14 @@ export const useApiDocuments = () => {
         ),
 
       streamDocumentFileApiCall: (mayanDocumentId: number, mayanFileId: number) =>
-        api.get<File>(`/documents/storage/${mayanDocumentId}/stream/${mayanFileId}`),
+        api.get<File>(`/documents/storage/${mayanDocumentId}/files/${mayanFileId}/stream`, {
+          responseType: 'blob',
```

Comment:
Streaming text documents and some other formats works fine without responseType: 'blob'. however, pdfs and word documents do not.